libs/client/repository/src/lib/dashboard/tags/tags.service.ts

```diff
@@ -39,8 +46,8 @@ export class TagsService {
 
     public createTag(kind: TagKind, form: TagsForm): Observable<TagsModel> {
         return this.network.createTag(kind, form).pipe(
-            tap(value => {
-                this.tagsStore.add(value);
+            tap(newTag => {
+                this.fetchTagsTrees(TagKind.Fandom).subscribe();
```

Comment:
Not super crazy about all these methods having a side-effect that updates the local tag cache, but I don't have a better solution off the top of my head, either.